index.js

```diff
@@ -0,0 +1,45 @@
+import Books from './modules/Books';
+
+const displayBook = document.querySelector('.display-books');
+const form = document.querySelector('form');
+
+// create an object
+const listBooks = new Books();
+function onPageReload() {
+  displayBook.innerHTML = listBooks.books
+    .map((book) => (
+  <div class='display'>
```

Comment:
- [ ] Please change this function to the arrow function to follow ES6 syntax.

Reply:
done